outgoing.go

```diff
@@ -54,21 +53,20 @@ func bounce(w http.ResponseWriter, sig, url string) {
 		return
 	}
 
-	safeUrl := html.EscapeString(url)
-	if safeUrl[:10] == "javascript" {
+	if url[:10] == "javascript" {
 		debug("URL starts with javascript url: %v", url)
 		errorResp(w)
 		return
 	}
-	if !strings.Contains(safeUrl, "://") {
-		safeUrl = "http://" + safeUrl
+	if !strings.Contains(url, "://") {
```

Comment:
That's fine with me. I only added that, because the original outgoing did it (https://svn.mozilla.org/projects/outgoing.mozilla.org/trunk/index.php)